Modules/Shop/Http/Services/ShopService.php

```diff
@@ -371,23 +368,23 @@ public function getProductFilters($request): array
                 }
                 break;
         }
-
+        echo  $allProductQuery;die;
         $products = $allProductQuery->get();
         if ($products) {
-            foreach ($products as $v) {
+            foreach ($products as $productDetails) {
```

Comment:
the variable name should be singular as it contains one product data, make it $productDetail.